lib/inferno/repositories.rb

```diff
@@ -3,6 +3,7 @@
 require_relative 'repositories/test_groups'
 require_relative 'repositories/test_suites'
 require_relative 'repositories/tests'
+require_relative 'repositories/validator_sessions'
```

Comment:
I'm not 100% sure which group this belongs with, but I don't think this needs to be require'd twice.